internal/fs/inode/file.go

```diff
@@ -597,11 +598,12 @@ func (f *FileInode) CacheEnsureContent(ctx context.Context) (err error) {
 
 func convertObjToMinObject(o *gcs.Object) (mo storage.MinObject) {
 	return storage.MinObject{
-		Name:           o.Name,
-		Size:           o.Size,
-		Generation:     o.Generation,
-		MetaGeneration: o.MetaGeneration,
-		Updated:        o.Updated,
-		Metadata:       o.Metadata,
+		Name:            o.Name,
+		Size:            o.Size,
+		Generation:      o.Generation,
+		MetaGeneration:  o.MetaGeneration,
+		Updated:         o.Updated,
+		Metadata:        o.Metadata,
```

Comment:
Should we also take care of /2 point mentioned in https://cloud.google.com/storage/docs/transcoding#good_practices ?